src/containers/org/members/components/RoleChipsList.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React, { useRef, useState } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import { Types } from 'lattice';
+// $FlowFixMe[missing-export]
+import { Chip } from 'lattice-ui-kit';
+import type { Role, UUID } from 'lattice';
+
+import RoleOverflowPopover from './RoleOverflowPopover';
+import usePriorityVisibility from './usePriorityVisibility';
+
+import { Routes } from '../../../../core/router';
+import { getUserProfile } from '../../../../utils';
+
+const { PermissionTypes } = Types;
+
+const ChipsList = styled.div`
+  display: flex;
+  flex: 1;
+  overflow: hidden;
+
+  > :not(:first-child) {
+    margin-left: 4px;
+  }
+`;
+
+type Props = {
+  currentRoleAuthorizations :Map;
+  member :Map;
+  onUnassign :(member :Map, role :Role) => void;
+  organizationId :UUID;
+  roles :Role[];
+};
+
+const RoleChipsList = ({
+  currentRoleAuthorizations,
+  member,
+  onUnassign,
+  organizationId,
+  roles,
+} :Props) => {
+  const chipListRef = useRef(null);
+  const [overflowAnchorEl, setOverflowAnchorEl] = useState(null);
+
+  const [priority, remainder] = usePriorityVisibility(chipListRef, roles, 4, 52);
+
+  const handleOverflowClick = (event) => {
+    setOverflowAnchorEl(event?.currentTarget);
+  };
+
+  const handleClose = () => {
+    setOverflowAnchorEl(null);
+  };
+
+  const { id } = getUserProfile(member);
+  const getHandleDelete = (role :Role) => (e :SyntheticEvent<HTMLElement>) => {
+    e.preventDefault();
+    handleClose();
+    onUnassign(member, role);
+  };
+
+  return (
+    <>
+      <ChipsList ref={chipListRef}>
+        {
+          priority.map((role, index) => {
+            const authorized = currentRoleAuthorizations.getIn([List(role.aclKey), PermissionTypes.OWNER], false);
```

Comment:
This might be a bug. `role.aclKey` is a regular array, i.e. `List<UUID[]>`. However, accessing a map with the acl key requires the key to be `List<List<UUID>>`, so I think this should be `List(List(role.aclKey))`.

Reply:
accessing the map with the aclKey takes List<UUID>, not a List<List<UUID>>. You're probably thinking one level too deep